packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -765,7 +769,7 @@ describe('SnapController', () => {
         [MOCK_SNAP_ID]: { version: DEFAULT_REQUESTED_SNAP_VERSION },
       }),
     ).rejects.toThrow(
-      'Cannot install snap "npm:@metamask/example-snap": The snap is not on the allowlist.',
+      'Failed to fetch snap "npm:@metamask/example-snap": Cannot install snap "npm:@metamask/example-snap": The snap is not on the allowlist.',
```

Comment:
This error is basically saying the same thing twice now? I would prefer just keeping the original message.

Reply:
It's being wrapped here: https://github.com/MetaMask/snaps/blob/main/packages/snaps-controllers/src/snaps/SnapController.ts#L2360-L2362

I can remove the "Cannot install snap "npm:@metamask/example-snap":" part from the registry implementation and just have the normal prefix?